packages/bodiless-organisms/src/components/Menu/asSubMenu.tsx

```diff
@@ -12,22 +12,18 @@
  * limitations under the License.
  */
 
-.rc-menu-submenu.bl-mega-menu ul.rc-menu-sub {
-  display: flex;
-}
-.rc-menu-submenu.bl-mega-menu ul.rc-menu-hidden {
-  display: none;
-}
+import { withDesign, stylable } from '@bodiless/fclasses';
+import { flow } from 'lodash';
+import { asTitledItem } from '@bodiless/components';
 
-.rc-tooltip.bl-menu-tooltip.rc-tooltip-placement-bottomLeft {
-  padding: 0;
-  opacity: 1;
-}
+const asSubMenu = flow(
```

Comment:
Let's see if we can get rid of this
- Making the sublist wrapper design elements stylable should be done to the base menu list (ie, it should be part of `asMenuSubList`.
- Why do we need to repeat `asTitledItem` here?